R/PrestoQuery.R

```diff
@@ -214,7 +224,8 @@ PrestoQuery <- setRefClass('PrestoQuery',
         'PrestoResult',
         statement = .statement,
         connection = .conn,
-        query = .self
+        query = .self,
+        bigint = .bigint %||% .conn@bigint
```

Comment:
I'd avoid using magrittr internally

Reply:
Do you mean the usage of `%>%`? It's only used in the `presto.fields.R` file to enhance readability. I'm indifferent to the usage of `%>%` in code, but if you strongly prefer to avoid it, I can change the code accordingly.

Reply:
Yea, I'm just not a huge fan. Adds a dependency that's not really needed.

Reply:
Understood. Done.